.idea/php-test-framework.xml

```diff
@@ -17,6 +17,7 @@
         <cache>
           <versions>
             <info id="Local" version="9.5.21" />
+            <info id="LocalH:\monorepo\vendor\autoload.php" version="10.4.2" />
```

Comment:
See https://youtrack.jetbrains.com/issue/WI-76125/PhpStorm-has-started-to-add-an-undesired-line-to-.idea-php-test-framework.xml-when-running-Pest-PHPUnit-tests

Must be reverted before merge. This was added in a temporary commit c639ccce8faef690d9729721d3a32315844dbc8a because I got tired of having to unstage the file which is changed each time I run the tests.